addons/grenades/functions/fnc_throwGrenade.sqf

```diff
@@ -36,17 +36,31 @@ if (local _unit) then {
         playSound3D [_file, objNull, false, getPosASL _projectile, _volume, _pitch, _distance];
     };
 
-    private _flashbangParams = (_config >> QGVAR(flashbang)) call BIS_fnc_getCfgDataArray;
-    private _nBangs = _flashbangParams param [0,0];
-    if (_nBangs > 0) then {
+    if (getNumber (_config >> QGVAR(flashbang)) == 1) then {
+        private _amount = 1;
+        if (isNumber (_config >> QGVAR(flashbangAmount))) then {
+            _amount = getNumber (_config >> QGVAR(flashbangAmount));
+        };
+
         private _fuzeTime = getNumber (_config >> "explosionTime");
-        private _std = _flashbangParams param [2,0.05];
+
+        private _standardDeviation = 0.05;
+        if (isNumber (_config >> QGVAR(flashbangDeviation))) then {
+            _standardDeviation = getNumber (_config >> QGVAR(flashbangDeviation));
```

Comment:
I don't think `flashbangDeviation` is especially ideal, as it could refer to inaccuracy in how it was thrown? Perhaps `flashbangIntervalStd` or `flashbangIntervalStandardDeviation`?

Reply:
flashbangIntervalDeviation preferably, or flashbangIntervalStandardDeviation if you insist.

Reply:
I called it `flashbangIntervalSpread` to make it clear I was being imprecise, and documented it more precisely in the wiki. I couldn't stomach `flashbangIntervalStandardDeviation`...